server/src/constants.js

```diff
@@ -0,0 +1,3 @@
+const distantFuture = 2147483647;
+
+export default distantFuture;
```

Comment:
eh probably don't need to make this default cause we expect to put other things here but don't worry for now